.ci/integrationTestSelector.groovy

```diff
@@ -137,7 +137,7 @@ pipeline {
         unstash "source"
         dir("${BASE_DIR}"){
           script {
-            docker.image('node:11').inside() {
+            docker.image('node:12').inside() {
```

Comment:
This is required from now on? I mean, `11` is not supported anymore?

Reply:
talking with Thomas the other day, told me even versions are the stables, and odd versions are develop versions, so let's use a stable one are develop versions